src/app/modules/ix-forms/components/ix-slide-toggle/ix-slide-toggle.component.ts

```diff
@@ -16,6 +16,7 @@ import { UntilDestroy } from '@ngneat/until-destroy';
 })
 export class IxSlideToggleComponent implements ControlValueAccessor {
   @Input() label: string;
+  @Input() labelPosition: 'before' | 'after' = 'after';
```

Comment:
I'd rather just use different order of things in reporting than add another input. This setting is going to be rarely used.